status-im/status-react
# feat: swap confirmation screen

fixes #20335 

### Summary

This PR implements Swap Confirmation screen, no backend integration yet and some data is hardcoded for now, but UI should be functional and most of the logic should be in place

<img src="https://github.com/status-im/status-mobile/assets/18485527/320350a6-f4e3-49ce-9f59-4f8b207e7773" width="300" />
<img src="https://github.com/status-im/status-mobile/assets/18485527/f93b43e5-dfd0-43a0-8517-5229b2263dce" width="300" />

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Log in
- Open wallet
- Select an account
- Tap on Swap
- Select an asset in asset to pay screen
- Tap on Swap confirmation button to navigate to Swap confirmation screen
- Verify UI looks as exepected

### Risk

Described potential risks and worst case scenarios.

Tick **one**:

status: ready
